tests/Nox.Types.Tests/Types/IpAddress/IpAddressTests.cs

```diff
@@ -3,9 +3,88 @@ namespace Nox.Types.Tests.Types;
 
 public class IpAddressTests
 {
+    [Theory]
+    [InlineData("0.0.0.0")]
+    [InlineData("10.0.0.0")]
+    [InlineData("127.0.0.1")]
+    [InlineData("172.16.0.0")]
+    [InlineData("192.0.2.1")]
+    [InlineData("255.255.255.255")]
+    public void Nox_IpAddress_Constructor_WithIPv4Value_ReturnsSameValue(string ipV4Value)
```

Comment:
Please remove the Nox prefix (we are starting doing this since is redundant)

Also see our guidelines that will be approved https://regusit.atlassian.net/wiki/spaces/NOX/pages/3153461266/Coding+Conventions+Styles+and+Guidelines and see this one: https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#best-practices its simple and defined in .Net ... 

```suggestion
    public void IpAddress_Constructor_WithIPv4Value_ReturnsSameValue(string ipV4Value)
```

Reply:
Thanks for the suggestions. Will apply them now.